staff/pedro-garriganogues/ecomerce/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+  <script src="main.js"></script>
+</head>
+
+<body>
+  <header id="header">
+    <img class="image" src="./images/seta.png" alt="404">
+    <input type="text" id="textHole" />
+    <img class="image" src="./images/lupa.png" alt="404">
+    <div id="info">
+      <div>Home</div>
```

Comment:
This would not be a correct use of div tag. Try to change it for another semantic.